src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
   putLetters    = letter <$> wordPutTiles wp
-  rackRemainder = fmap fromLetter $ foldl' (flip delete) rackLetters putLetters
+  rackRemainder = fmap fromLetter $
+                  applyNTimes blanksUsed (delete Blank) $
+                  foldl' (flip delete) rackLetters putLetters
+  applyNTimes n f = foldr (.) id (replicate n f)
```

Comment:
Oh. I didn't notice that. Interesting; I don't think we should do that. If a user wants to play a blank tile, then ... their problem. Right?